openwisp_controller/config/migrations/0017_default_organization_config_settings.py

```diff
@@ -19,7 +19,7 @@ def make_default_config_settings_organization(apps, schema_editor):
 class Migration(migrations.Migration):
     dependencies = [
        ('openwisp_users', '0003_default_organization'),
-       ('config', '0015_default_groups_permissions'),
+       ('config', '0016_auto_20181208_0622'),

```

Comment:
please rename this migration so that it has a human readable name, eg: `0016_template_unique_together_name_org.py`.

@atb00ker seems we don't have the migration name check in this repo, I thought we had that check in all our repos now.. but I was wrong, will ping you via chat about it